homeassistant/components/dreo/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "dreo",
+  "name": "Dreo",
+  "codeowners": ["@dreo-team"],
```

Comment:
Please add yourself as code owner.
Also `dreo-team` need to comment on this PR and approve that they are willing to be the code owner